config.json

```
@@ -6,6 +6,7 @@
 		"method_allowed": {
 			"/uploads" : 7,
 			"/index.html" : 1,
+			"/input.php" : 1,
```

Should we add `/cgi/` ?
```suggestion
			"/cgi/input.php" : 1,
```

---

yes